### DISCUSSION TITLE
Provide default style to ease adoption of ZagJS

### DISCUSSION THREAD

**Discussion by samuelstroschein** [chakra-ui/zag#485] [Ideas]
The adoption barrier for Zag.js can be (substantially?) reduced if Zag.js provides a default style/components that can be incrementally modified as needed and desired. 

https://ui.shadcn.com/ is an example of default radix UI + tailwind CSS styles.

**Comment by anubra266** [comment]
@samuelstroschein check out github.com/anubra266/arkxamples/

**Reply by samuelstroschein** [reply]
Fixed link https://github.com/anubra266/arkxamples/

**Comment by segunadebayo** [comment]
Closing — Zag is headless by design and does not ship default styles. Component libraries like Ark UI and Chakra UI provide the styled layer on top.
